ipn/ipnlocal/c2n.go

```diff
@@ -121,6 +121,14 @@ func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		writeJSON(w, res)
+	case "/appconnector/routes":
+		switch r.Method {
+		case httpm.GET:
+			b.handleC2NAppConnectorDomainRoutesGet(w, r)
```

Comment:
Do you mean that I'm doing the following?

```
b.handleC2NAppConnectorDomainRoutesGet(w, r)
```

and 

```
http.Error(w, "bad method", http.StatusMethodNotAllowed)
return
```
If so, that is the same pattern as `/update`, `/posture/identity`. Others do it differently.

Reply:
Here it doesn't seem to matter, since the fall-off at the end is empty anyway. That said, I'd probably put in an explicit return anyway so the reader doesn't have to figure that out each time. YMMV.